.github/workflows/perf-regression-ci.yml

```diff
@@ -97,18 +97,25 @@ jobs:
   openjdk-perf-regression:
     runs-on: [self-hosted, Linux, freq-scaling-off]
     timeout-minutes: 1440
+    env:
+      # This version will be used as the canary version, and will be used to checkout both
```

Comment:
Personally I think we can run this version forever unless there are things preventing us from doing so.  For example, the toolchain required for building the canary version is no longer available.  This is not likely to happen, but if that happens, we may bump the canary version.

Reply:
Sounds good. Can you write the above into the comments?

Also can you update the doc in https://github.com/mmtk/mmtk-core/pull/1206 to include a short description about canary? We can either merge that PR first, then you update it in this PR. Or you directly update the PR and merge it.

Reply:
Yes.  I'll write that in to the comment, and also change the ci-perf-kit version to 0.8.1.

I will merge #1206, too, but I prefer it merged AFTER this one because I think it is better to *have* the canary and then describe what that is.  By the way, this should be a good chance to take advantage of our "merge queue" mechanism to see if we can merge both at the same time.

Reply:
Merge queue should be working fine. But for this case, I doubt if you can merge these two PRs with the merge queue, as they would have conflict on the same file `docs/team/ci.md`.

Reply:
Sorry for the typo.  I meant I will make changes to #1206 to mention "canary", and then we use the merge queue to merge #1209 (this PR) and #1206.  Since this PR only modifies `perf-regression-ci.yml` and #1206 only modified `ci.md`, there should be no conflict.